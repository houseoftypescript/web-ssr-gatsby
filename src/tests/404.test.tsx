import { render } from '@testing-library/react';
import { PageProps } from 'gatsby';
import React from 'react';
import NotFoundPage from '../pages/404';

describe('NotFoundPage', () => {
  it('to match snapshot', () => {
    const { container } = render(<NotFoundPage {...({} as PageProps)} />);
    expect(container).toMatchSnapshot();
  });
});
