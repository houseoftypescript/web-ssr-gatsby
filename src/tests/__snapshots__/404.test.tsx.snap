// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage to match snapshot 1`] = `
<div>
  <main
    class="w-full h-screen flex flex-col items-center justify-center gap-4"
  >
    <h1
      class="text-9xl font-bold"
    >
      404
    </h1>
    <p
      class="text-3xl uppercase"
    >
      Page Not Found
    </p>
  </main>
</div>
`;
